l10n_ch_fds_postfinance/models/fds_postfinance_account.py

```diff
@@ -103,10 +103,13 @@ def verify_directories_button(self):
                 key[0].private_key_crypted)
             key_pass = self.authentication_key_ids.config()
 
+            cnopts = pysftp.CnOpts(knownhosts='/etc/ssh/ssh_known_hosts')
```

Comment:
I fear this will fail on a non-linux host.

Reply:
Hello. As I mentioned I use a system user. No Home folder, so no usefullness into connecting once.

Please park this for a few days, i am tryng to solve this with pysftp directly. My issue was recongnised as a bug there ..